ros1_cpptemplate/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+##############################################################################
+# CMake
+##############################################################################
+
+cmake_minimum_required(VERSION 2.8.3)
+project(ros1_cpptemplate)
+
+##############################################################################
+# Dependencies
+##############################################################################
+
+find_package(catkin REQUIRED
+            COMPONENTS
+              roslint
+              rosconsole
+)
+
+##############################################################################
+# Catkin
+##############################################################################
+
+catkin_package(
```

Comment:
How about having the cpptemplate without catkin, using pure cmake stuff, to show pure c++ integration in ROS ?